src/arrays/multidimensional_arrays/hard/q6/MainTest.java

```diff
@@ -0,0 +1,37 @@
+package arrays.multidimensional_arrays.hard.q6;
+
+import global.BaseTest;
+import global.variables.Clause;
+import global.variables.clauses.NewLine;
+import global.variables.clauses.StringLiteral;
+import global.variables.wrappers.Optional;
+
+public class MainTest extends BaseTest {
+    // Parsons
+
+    public static final int[][] nums = {{4, 5, 6}, {66, 54, 21}, {609, 998, 543}};
+
+    public Clause[] testSentence() {
+        int[] ans = answer();
+        int s1 = ans[0];
+        int s2 = ans[1];
+        return new Clause[]{
+                new StringLiteral("Sum of diagonal 1  = " + s1 + ", Sum of diagonal 2 = " + s2)
```

Comment:
Consider changing this to be StringLiteral IntegerLiteral StringLiteral Integer Literal with a test method so the error message can be more specific which number is wrong.